test/unit/nfts/seasons/ERC1155SeasonOne.t.sol

```diff
@@ -203,4 +204,62 @@ contract UnitTestERC1155SeasonOne is SeasonBase {
         assertEq(token.balanceOf(address(this)), 400);
         assertEq(token.balanceOf(address(_seasonOne)), beforeTotalRewardTokens - 400);
     }
+
+    function testRedeemWhenPaused() public {
```

Comment:
Should also add to the integration tests for pausing the entire system.

Reply:
yep, new PR